src/references/ethereum-units.json

```diff
@@ -1,8 +1,8 @@
 {
   "basic_approval": 55000,
   "basic_swap": 200000,
-  "basic_swap_arbitrum": 3500000,
```

Comment:
these seem like they should be reverted since the l1 gas shit was the issue yeah? otherwise just wanna make sure these numbers are good/accurate